src/components/select/select.js

```diff
@@ -288,19 +288,16 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
             }
           });
 
-        // Wait until postDigest so that we attach after ngModel's
-        // blur listener so we can set untouched.
-        $mdUtil.nextTick(function () {
-          element.on('blur', function() {
-            if (untouched) {
-              untouched = false;
-              ngModelCtrl.$setUntouched();
-            }
 
-            if (selectScope.isOpen) return;
-            containerCtrl && containerCtrl.setFocused(false);
-            inputCheckValue();
-          });
+        element.on('blur', function() {
```

Comment:
Yes, commenting the whole handler will indeed make those tests fail. I meant, that the change to $evalAsync is not needed. In later point of time, when user enables disabled mdSelect, everything should work as before. Problems with blur on mdSelect will be fixed in https://github.com/angular/material/pull/7646. If you want to include those changes in your PR, as we talked about, then please include it as it is there, including the test and condition on "isOpen".